src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,310 @@
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}-visual.component.ts</stache-code>: The TypeScript code for the component to render and take a screenshot of.
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}.visual-spec.ts</stache-code>: The Protractor code to run the screenshot tests.
+    </li>
+    <li>
+      <stache-code>index.html</stache-code>: The route file that contians the component to render and take a screenshot of.
+    </li>
+  </ul>
+
+  <p>
+    To run visual regression tests locally, you must first run them locally against the <stache-code>master</stache-code> branch and then run the tests against your feature branch. Keep in mind that you need to use the same monitor and resolution on both set of tests. If your visual regression tests fail, you can check the failed tests at <stache-code>skyux2\skyux-spa-visual-tests\screenshots-diff-local\diff</stache-code> to find the cause of the failures.
+  </p>
+
+  <stache-page-anchor>
+    Submit code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Commit and push your changes to the repo.
+    </li>
+    <li>
+      Submit a pull request.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Available NPM scripts
+  </stache-page-anchor>
+
+  <p>
+    <stache-code>npm install</stache-code> and <stache-code>npm start</stache-code> are special commands. All other commands listed below should use the <stache-code>npm run COMMAND</stache-code> format. For example, <stache-code>npm run build</stache-code>.
+  </p>
+
+  <table class="content-table">
+    <thead>
+      <tr style="text-align: left;" class="sky-emphasized">
+        <th style="width: 18em;">Script</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><stache-code>build</stache-code></td>
+        <td>Cleans the previous build, compiles the Sass, and transpiles the JavaScript.</td>
+      </tr>
+      <tr>
+        <td><stache-code>clean</stache-code></td>
+        <td>Cleans the previous build.</td>
+      </tr>
+      <tr>
+        <td><stache-code>clean:full</stache-code></td>
+        <td>	Cleans the previous build, <stache-code>node_modules</stache-code>, and coverage reports..</td>
+      </tr>
+      <tr>
+        <td><stache-code>lint</stache-code></td>
+        <td>Runs TypeScript linter.</td>
+      </tr>
+      <tr>
+        <td><stache-code>test</stache-code></td>
+        <td>Runs unit tests and visual regression tests.</td>
+      </tr>
+      <tr>
+        <td><stache-code>test:unit</stache-code></td>
+        <td>Runs Karma unit tests.</td>
+      </tr>
+      <tr>
+        <td><stache-code>test:visual</stache-code></td>
+        <td>Runs WebDriver visual regression tests and accessibility tests.</td>
+      </tr>
+      <tr>
+        <td><stache-code>watch</stache-code></td>
+        <td>Runs Karma unit tests and watches for file changes.</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <stache-page-anchor>
+    Report bugs
```

Comment:
Do you think we need this section here when we already have it on the parent page? We also have the Recommended tools section on the parent page, and you didn't copy that over even though it's currently in the CONTRIBUTING.md file, so that makes me think maybe we don't need this section here. That said, I can see an argument for including this section here, but if we do that, I'd create an includes file to keep this version and the version on the parent page in sync.

Reply:
I'm with you in that I don't think we need the Report bugs section either. I removed it.

Reply:
Sounds good. Resolving the conversation...